### PR TITLE
Add note about blank payloads

### PR DESCRIPTION
**Pull request by dthaler**
Based on discussion in issue #495

### CODE REVIEW

**Review by tychonievich — APPROVED** [review]

File: specification/gedcom-3-structures-1-organization.md [hunk 1]
```diff
@@ -1138,6 +1138,14 @@ A place, which can be represented in several ways:
     or in the `HEAD`.`PLAC`.`FORM` if there is no `FORM` substructure.
     If neither `FORM` exists, the meaning of the elements are not defined in this specification beyond being names of jurisdictions of some kind, ordered from smallest to largest.
 
+:::note
+    Existing applications often default to assuming a `FORM` of "City, County, State, Country",
+    and may even ignore any `FORM` substructures and treat payloads with a smaller number of
```
Comment (dthaler) [inline specification/gedcom-3-structures-1-organization.md:1143]:
```suggestion
    and some even ignore any `FORM` substructures and treat payloads with a smaller number of
```

File: specification/gedcom-3-structures-1-organization.md [hunk 2]
```diff
@@ -1138,6 +1138,14 @@ A place, which can be represented in several ways:
     or in the `HEAD`.`PLAC`.`FORM` if there is no `FORM` substructure.
     If neither `FORM` exists, the meaning of the elements are not defined in this specification beyond being names of jurisdictions of some kind, ordered from smallest to largest.
 
+:::note
+    Existing applications often default to assuming a `FORM` of "City, County, State, Country",
+    and some even ignore any `FORM` substructures and treat payloads with a smaller number of
+    elements as if they had additional blank elements at the end. Accordingly, to increase
+    the chances of interoperability it is recommended that blank elements be used to ensure
+    payloads can be interpreted as best as possible in such a circumstance.
+:::
```
Comment (dthaler) [inline specification/gedcom-3-structures-1-organization.md:1147]:
```suggestion
    Existing applications often default to assuming a `FORM` of "City, County, State, Country",
    and some even ignore any `FORM` substructures and treat payloads with a smaller number of
    elements as if they had additional blank elements at the end. Accordingly, to increase
    the chances of interoperability it is recommended that blank elements be used to ensure
    payloads can be interpreted as best as possible in such a circumstance.
```

File: specification/gedcom-3-structures-1-organization.md [hunk 3]
```diff
@@ -1138,6 +1138,14 @@ A place, which can be represented in several ways:
     or in the `HEAD`.`PLAC`.`FORM` if there is no `FORM` substructure.
     If neither `FORM` exists, the meaning of the elements are not defined in this specification beyond being names of jurisdictions of some kind, ordered from smallest to largest.
 
+:::note
+    Existing applications often default to assuming a `FORM` of "City, County, State, Country",
+    and some even ignore any `FORM` substructures and treat payloads with a smaller number of
+    elements as if they had additional blank elements at the end. Accordingly, to increase
+    the chances of interoperability it is recommended that blank elements be used to ensure
```
Comment (dthaler) [inline specification/gedcom-3-structures-1-organization.md:1145]:
```suggestion
    the chances of interoperability an application might want to use blank elements to ensure
```

File: specification/gedcom-3-structures-1-organization.md [hunk 4]
```diff
@@ -1138,6 +1138,14 @@ A place, which can be represented in several ways:
...
+    payloads can be interpreted as best as possible in such a circumstance.
```
Comment (dthaler) [inline specification/gedcom-3-structures-1-organization.md:1146]:
```suggestion
    Some applications or users have defaulted to assuming a `FORM` of "City, County, State, Country",
    and some even ignore any `FORM` substructures and treat payloads with a smaller number of
    elements as if they had additional blank elements at the end.
```

**Comment by dthaler** [comment]
Approved during GEDCOM Steering Committee meeting

**Pull request merged**